### PR TITLE
[Merged by Bors] - Add community checkpoint sync endpoints to book

### PR DESCRIPTION
**Pull request by remyroy**
## Proposed Changes

Add a section on the new community checkpoint sync endpoints in the book. This should help stakers sync faster even without having to create an account.

### CODE REVIEW

**Review by michaelsproul — APPROVED** [review]
Niiice!

**Comment by coincashew** [comment]
Just a suggestion. The` /` has caused issues for some CL clients.

recommend 
--checkpoint-sync-url=https://beaconstate.info

File: book/src/checkpoint-sync.md [hunk 1]
```diff
@@ -41,6 +41,13 @@ Once the checkpoint is loaded Lighthouse will sync forwards to the head of the c
 If a validator client is connected to the node then it will be able to start completing its duties
 as soon as forwards sync completes.
 
+### Use a community checkpoint sync endpoint
+
+The Ethereum community provides various [public endpoints](https://eth-clients.github.io/checkpoint-sync-endpoints/) for you to choose from for your initial checkpoint state. Select one for your network and use it as the url for the `--checkpoint-sync-url` flag.  e.g.
+```
+lighthouse bn --checkpoint-sync-url https://beaconstate.info/ ...
```
Comment (InvisibleSymbol) [inline book/src/checkpoint-sync.md:48]: I feel like People might just end up copying this line out of laziness and all end up using this specific endpoint, which is sub-optimal from a safety perspective (no single checkpoint endpoint should be recommended). How about using a non-working domain instead, like `example.com`?

Comment (michaelsproul) [reply book/src/checkpoint-sync.md:48]: that's a good point, I'm happy to merge that change if you or @remyroy would like to PR it (assuming no objections?)

Comment (remyroy) [reply book/src/checkpoint-sync.md:48]: I'm good with @InvisibleSymbol suggestion. Let me do another PR.

**Pull request closed without merging**